extensions/cornerstone-dicom-sr/src/utils/getFilteredCornerstoneToolState.ts

```diff
@@ -1,17 +1,10 @@
-import OHIF from '@ohif/core';
 import { annotation } from '@cornerstonejs/tools';
-const { log } = OHIF;
 
 function getFilteredCornerstoneToolState(measurementData, additionalFindingTypes) {
   const filteredToolState = {};
 
   function addToFilteredToolState(annotation, toolType) {
-    if (!annotation.metadata?.referencedImageId) {
-      log.warn(`[DICOMSR] No referencedImageId found for ${toolType} ${annotation.id}`);
-      return;
-    }
-
-    const imageId = annotation.metadata.referencedImageId;
+    const imageId = annotation.metadata?.referencedImageId ?? 'none';
```

Comment:
Do you want to use 'none' string here? its considered a valid value so it wont be catched by !imageId conditionals below.

Reply:
Add a comment to that to indicate why you are doing this.

Reply:
Still awaiting a comment - adding a 'none' key is unusual so it would be nice to know why.